pandas/core/resample.py

```diff
@@ -1266,7 +1269,7 @@ def get_resampler(obj, kind=None, **kwds):
     """
     Create a TimeGrouper and return our resampler.
     """
-    tg = TimeGrouper(**kwds)
+    tg = TimeGrouper(**kwds, _warning_stack_level=2)
```

Comment:
this is very odd to do, don't

Reply:
Yep! I agree, I changed that in the last commit, see https://github.com/pandas-dev/pandas/pull/31809#discussion_r379860446 for more context.